pkg/trace/api/api.go

```diff
@@ -494,6 +494,9 @@ func decodeTracerPayload(v Version, req *http.Request, cIDProvider IDProvider, l
 		}
 		var traces pb.Traces
 		err = traces.UnmarshalMsgDictionary(buf.Bytes())
+		if err != nil {
+			return nil, err
+		}
```

Comment:
nit: consider inlining the conditional with
```go
if err = traces.UnmarshalMsgDictionary(buf.Bytes()); err != nil {
    return nil, err
}
```

Reply:
Done, thanks